packages/api-client/src/granules.ts

```diff
@@ -809,3 +812,37 @@ export const bulkOperation = async (params: {
     expectedStatusCodes: 202,
   });
 };
+
+/**
+ * Bulk Granule Operations
+ * POST /granules/bulkChangeCollection
+ */
```

Comment:
Do we have a breakdown of the params somewhere?

Reply:
Yeah, there's a good docstring in the endpoint. I don't think we want to repeat that info here. Nevermind. Nothing to see here.